crates/iceberg/src/metadata_scan.rs

```diff
@@ -50,6 +52,13 @@ impl MetadataTable {
         }
     }
 
+    /// Get the manifests table.
+    pub fn manifests(&self) -> ManifestsTable {
+        ManifestsTable {
+            metadata_table: self,
```

Comment:
Hi, I think we can simply use `Table` here, which suggests that `MetadataTable` is merely a wrapper and doesn't implement any additional API.